### PR TITLE
New package: Code::blocks

### LINKED ISSUE #11151: [Package]: Code blocks
### Package description

Code blocks light weight  IDE.

### Home page URL

https://www.codeblocks.org/

### Source code URL

http://www.codeblocks.org/downloads/source/

### Packaging policy acknowledgement

- [X] I have read and understand the [Packaging Policy](https://github.com/termux/termux-packages/blob/master/CONTRIBUTING.md#packaging-policy).

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by ifurther**
Close #11151 .

### CODE REVIEW

File: x11-packages/codeblocks/build.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+TERMUX_PKG_HOMEPAGE=https://www.codeblocks.org/
+TERMUX_PKG_DESCRIPTION="Code::Blocks is the open-source, cross-platform Integrated Development
+ Environment (IDE)."
```
Comment (Grimler91) [inline x11-packages/codeblocks/build.sh:3]: Please write description on a single line

**Review by Grimler91 — CHANGES_REQUESTED** [review]
Please squash the commits and rebase to get rid of the merge commits

**Review by Grimler91 — CHANGES_REQUESTED** [review]
Please use tabs for indentation, as outlined in the [coding guidelines](https://github.com/termux/termux-packages/wiki/Coding-guideline)

File: x11-packages/codeblocks/build.sh (hunk 1 above)
Comment (ifurther) [reply x11-packages/codeblocks/build.sh]: Sorry, I fixed.

**Review by Grimler91 — CHANGES_REQUESTED** [review]
Please rebase to get rid of the merge commits, and re-write the commit messages `add patch file for wxWidgets3.16` and `disable compiler` so that it is easier to se what package they affect, for example by prefixing `codeblocks:`

**Comment by xtkoba** [comment]
Cleanup will be done in #14429.

**Pull request closed without merging**